front/app/components/Form/formConfig.tsx

```diff
@@ -75,30 +75,26 @@ const commonRenderers = [
 ];
 
 export const selectRenderers = (formType: 'default' | 'input' | 'survey') => {
-  switch (formType) {
-    case 'default':
-      return commonRenderers;
-    case 'survey':
-    case 'input':
-      return [
-        { tester: linearScaleControlTester, renderer: LinearScaleControl },
-        {
-          tester: multiSelectCheckboxControlTester,
-          renderer: MultiSelectCheckboxControl,
-        },
-        {
-          tester: singleSelectRadioControlTester,
-          renderer: SingleSelectRadioControl,
-        },
-        {
-          tester: singleSelectRadioEnumControlTester,
-          renderer: SingleSelectRadioEnumControl,
-        },
-        {
-          tester: singleAttachmentControlTester,
-          renderer: SingleAttachmentControl,
-        },
-        ...commonRenderers,
-      ];
-  }
+  if (formType === 'default') return commonRenderers;
```

Comment:
Changes to this file are unrelated to anything, was just bothered by the weird use of the `switch` statement